src/background.js

```diff
@@ -133,7 +133,7 @@ const messageOps = {
     delete message.tabId;
     toSave[idForItem(message)] = message;
     return browser.tabs.query({}).then(tabs => {
-      if (tabs.length <= 1) {
+      if (tabs.length < 1) {
```

Comment:
It's `<= 1` because we don't remove the tab until down below, so if there's only one tab, and you snooze it, we won't open an `about:home` tab, and you'll be left with nothing.